core/src/main/java/org/arquillian/smart/testing/hub/storage/local/FileSystemOperations.java

```diff
@@ -62,7 +62,7 @@ static void deleteFile(Path path, boolean catchException) {
         }
     }
 
-    static void deleteDirectory(Path directory, boolean catchException) {
+    public static void deleteDirectory(Path directory, boolean catchException) {
```

Comment:
doesn't have to be public - see my comment: https://github.com/arquillian/smart-testing/pull/183/files#r140523705